SPECS/systemd/systemd.spec

```diff
@@ -12,6 +12,8 @@ Source1:        50-security-hardening.conf
 Source2:        systemd.cfg
 Source3:        99-dhcp-en.network
 Patch0:         fix-journald-audit-logging.patch
+# Patch for skipping the tests: test-mountpoint-util, test-mount-util, test-fileio, test-fd-util, test-repart - Some mounts are failing in chroot
```

Comment:
We need to bump the `Release` if we change anything in the spec.
Also, a corresponding changelog needs to be added as well :)